cdap-data-fabric/src/main/java/co/cask/cdap/data/runtime/DataSetsModules.java

```diff
@@ -119,7 +119,7 @@ protected void configure() {
 
   @Override
   public Module getDistributedModules() {
-    return getDistributedModules(false);
+    return getDistributedModules(true);
```

Comment:
If it is always controlled by cConf, then why need to pass in a boolean in this case?